static/src/js/actions/admin/retrievals.js

```diff
@@ -156,3 +157,40 @@ export const updateAdminRetrievalsPageNum = (pageNum) => (dispatch) => {
 
   dispatch(actions.fetchAdminRetrievals())
 }
+
+/**
+ * Sends a request to have the provided order requeued for processing
+ * @param {integer} orderId
+ */
+export const requeueOrder = (orderId) => (dispatch, getState) => {
+  const state = getState()
+
+  // Retrieve data from Redux using selectors
+  const earthdataEnvironment = getEarthdataEnvironment(state)
+
+  const { authToken } = state
+
+  try {
```

Comment:
Is this try-catch necessary if the requeue order command has a catch in the promise chain?

Reply:
Good catch, not sure why that was there in the function I copied from 🤷‍♂️